cli/src/pcluster/models/cluster_config.py

```diff
@@ -164,6 +171,7 @@ class SharedStorageType(Enum):
     """Define storage types to be used as shared storage."""
 
     EBS = "ebs"
+    RAID = "raid"
     EFS = "efs"
     FSX = "fsx"
```

Comment:
The enum can be used for different things.
As you can see in the code we have the same Class `SharedEbs` to manage both Raid and EBS because they have the same attributes, the raid configuration is an optional setting, so by using the type attribute we can avoid defining another class for SharedRaid.

Then in CDK template generation we're using the Enum type as a keyword to store information about the shared storage ids in a map. We need to distinguish between raid ids and ebs volume ids and thanks to the enum we can easily have a key to be used in dictionaries without the need to remember the specific key to use, avoiding typos.

By having an enum we can iteract on them and it's what we're doing to create the shared_storage_ids dictionary in cdk template generation.